profiler/include/profiler/profile_grouped_gemm_fixed_nk_impl.hpp

```diff
@@ -88,11 +88,12 @@ bool profile_grouped_gemm_fixed_nk_impl(int do_verification,
 
         c_m_n_host_results.push_back(
```

Comment:
Well, I think the current debugging output was added by different developers at different times without considering a general uniform approach. 
I think we should eventually implement something similar to what MIOpen is using, add a logging class and set different levels of logging to errors, warnings, debugging, etc.
For now I just wanted to get rid of the macros and enable the use of environment variable. I didn't modify any of the existing print statements since I'm not sure who may still be using them.